tests/find-package-path-test.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+var assert = require('assert');
+var findPackagePath = require('../lib/find-package-path');
+
+describe.skip('findPackagePath', function() {
```

Comment:
@dcombslinkedin can you implement these tests

Reply:
Done (to the point I can resolve this conversation).